src/content/reference/react/useImperativeHandle.md

```diff
@@ -166,9 +166,9 @@ input {
 
 ---
 
-### Exposing your own imperative methods {/*exposing-your-own-imperative-methods*/}
+### Mengekspos methods imperatif Anda sendiri {/*exposing-your-own-imperative-methods*/}
 
-The methods you expose via an imperative handle don't have to match the DOM methods exactly. For example, this `Post` component exposes a `scrollAndFocusAddComment` method via an imperative handle. This lets the parent `Page` scroll the list of comments *and* focus the input field when you click the button:
+*Methods* yang diekspos melalui penanganan imperatif tidak harus sesuai dengan *DOM Method* secara persis. Sebagai contoh, komponen `Post` berikut mengekspos sebuah *method* `scrollAndFocusAddComment` melalui penanganan imperatif. Hal ini memungkinkan induk `Page` menggulir daftar komentar *dan* memfokuskan bidang input ketika Anda mengklik tombol:
```

Comment:
```suggestion
*Methods* yang diekspos melalui penanganan imperatif tidak harus sesuai dengan *DOM methods* secara persis. Sebagai contoh, komponen `Post` berikut mengekspos sebuah *method* `scrollAndFocusAddComment` melalui penanganan imperatif. Hal ini memungkinkan induk `Page` menggulir (*scroll*) daftar komentar *dan* memfokuskan bidang input ketika Anda mengklik tombol:
```

1. Retain the original `DOM methods` plural form.
2. Introduce `scroll` translation for the first time. Let's propose it here: https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5772140.